conans/model/conan_file.py

```diff
@@ -147,9 +147,6 @@ def __init__(self, output, runner, settings, user=None, channel=None, local=None
         # Keys are the package names, and the values a dict with the vars
         self.deps_user_info = DepsUserInfo()
 
-        self.copy = None  # initialized at runtime
```

Comment:
Fields assigned in `__init__` override ones defined in `locals` dictionary (i.e. coming from class definition) and thus pylint complains about `self.copy` is not being callable (since it's a `NoneType`).

I tried to work this around by overriding inference of call to `.copy()` but it looked too complicated.